pages.it/common/alacritty.md

```diff
@@ -0,0 +1,24 @@
+# alacritty
+
+> Multipiattaforma, GPU-accelerato emulatore di terminale.
+> Maggiori Informazioni: <https://github.com/alacritty/alacritty>.
+
+- Apri un nuovo caso di Alacritty:
+
+`alacritty`
+
+- Esegui in una posizione specifica:
```

Comment:
`posizione` is `directory`? I think not.
```suggestion
- Esegui in una cartella specifica:
```

Reply:
I've seen directory used in Italian, but cartella would be the approximate Italian translation